frontend/src/e2e-playwright/specs/5_employee/unit-groups.spec.ts

```diff
@@ -184,7 +204,25 @@ describe('Unit groups - staff', () => {
   })
 
   test('Staff will not see children with a missing group placement', async () => {
-    const groupsSection = await loadUnitGroupsPage()
-    await groupsSection.missingPlacementsSection.assertRowCount(0)
+    const groupsPage = await loadUnitGroupsPage()
+    await groupsPage.missingPlacementsSection.assertRowCount(0)
+  })
+
+  test('Staff does not see child occupancy factor', async () => {
+    await Fixture.groupPlacement()
+      .with({
+        daycareGroupId: groupId,
+        daycarePlacementId: child1DaycarePlacementId,
+        startDate: placementStartDate.format('yyyy-MM-dd'),
+        endDate: placementEndDate.format('yyyy-MM-dd')
+      })
+      .save()
+
+    const groupsPage = await loadUnitGroupsPage()
+    await groupsPage.childOccupancyFactorColumnHeading.waitUntilHidden()
+    await waitUntilEqual(
+      () => groupsPage.childOccupancyFactorColumnData.count(),
+      0
+    )
```

Comment:
I think it would be a good idea to abstract this behind a single `groupsPage` method.